utils/db_utils.py

```diff
@@ -98,6 +100,22 @@ def query_confirmed_trips(start_date, end_date):
     # logging.debug("After filtering, the actual data is %s" % df.head().trip_start_time_str)
     return df
 
+def query_demographics():
+    ts = esta.TimeSeries.get_aggregate_time_series()
+    
+    entries = ts.find_entries(
+        key_list=["manual/demographic_survey"],
+    )
+
+    df = pd.json_normalize(list(entries))
+
```

Comment:
I am not going to hold this up but have logged a potential improvement for later.

Reply:
Potential improvement logged here:
https://github.com/e-mission/op-admin-dashboard/issues/51#issuecomment-1709275660